dts/bindings/sensor/we,wsen-hids-2525020210002.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2024 Würth Elektronik eiSos GmbH & Co. KG
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    Würth Elektronik WSEN-HIDS-2525020210002 humidity sensor
+
+compatible: "we,wsen-hids-2525020210002"
+
+include: [sensor-device.yaml, i2c-device.yaml]
+
+properties:
+
+  precision:
+    type: string
+    default: "Medium"
+    enum:
+      - "Low"
+      - "Medium"
+      - "High"
```

Comment:
I looked over dts binding properties and it appears the convention for this is: `low, mid, high`. I suggest we follow that.

Reply:
i made the modifications any idea where i can find these conventions, this would be relevant for the next drivers im going to add.

Reply:
I found it by searching on existing bindings under the same `dts/bindings/**`. Here's some examples:

https://github.com/zephyrproject-rtos/zephyr/blob/6f937c9ceaf67397189fd2e1275ff8eba264c801/dts/bindings/sensor/nxp%2Clpcmp.yaml#L46-L49

https://github.com/zephyrproject-rtos/zephyr/blob/6f937c9ceaf67397189fd2e1275ff8eba264c801/dts/bindings/clock/nordic%2Cnrf-auxpll.yaml#L88-L90

There doesn't seem to be a standardized convention but it's best to replicate existing patterns.